sqlite-fts5/build.gradle

```diff
@@ -48,12 +44,12 @@ android {
             path 'src/main/CMakeLists.txt'
         }
     }
-    buildToolsVersion '26.0.2'
+    buildToolsVersion ANDROID_BUILD_TOOLS_VERSION
```

Comment:
missed toInteger()?

Reply:
this one actually expects a string, but if I get the ext stuff work, then I think I can remove the `.toInteger` stuff